executor/transaction_processor.go

```diff
@@ -644,7 +644,7 @@ func (a *toscaTxContext) EmitLog(log tosca.Log) {
 
 func (a *toscaTxContext) GetLogs() []tosca.Log {
 	res := []tosca.Log{}
-	for _, l := range a.db.GetLogs(common.Hash{}, 0, common.Hash{}) {
+	for _, l := range a.db.GetLogs(common.Hash{}, 0, common.Hash{}, 0) {
```

Comment:
Instead of `0` you should probably pass the time stamp from `a.blockEnvironment`; Wdyt?

While the timestamp is ignored in Tosca, the difference will show up in DB logs when comparing different processor implementations.